blocks/large-promo-block/features/large-promo/_children/extractVideoEmbedFromStory.js

```diff
@@ -0,0 +1,12 @@
+const extractVideoEmbedFromStory = (content) => {
```

Comment:
@vinnie-olsen Could we add this as a helper function since multiple promo blocks seem to be using it?

Reply:
@beltrancaliz that was actually my original plan but I didn't see anywhere in the repo where globally-used helper functions would live. I also noticed that other blocks were redefining helper functions used by multiple blocks so I just assumed that was protocol here since blocks are sort of 'self-contained'.

Is there a global "helpers" block in the repo that we are supposed to put helper functions used by multiple blocks in?

Reply:
@visionbegin do you have any thoughts on this? I'm trying to figure out how to proceed or if this is acceptable since the promo blocks are already doing duplication of helper functions.

Reply:
@vinnie-olsen I think a good approach would be to place the helper in engine-sdk.

Reply:
@visionbegin added the helper function to `engine-theme-sdk` 🙂  PR/branch to test with locally here: https://github.com/WPMedia/engine-theme-sdk/pull/135